### ISSUE TITLE
[CULT] BOTA-SC : rubrique propagation grisée

### ISSUE THREAD

**Issue by cibotalista** [Computaceae-IT/cit-ticketing#1232]
Url : https://jbp.botalista.community/#/cultivated/view/58379
Username : gomes lemos

Description : Bonjour les DEV de Botalista, 

Un utilisateur souhaite renseigner la rubrique propagation d'un SC cultivé vivant n° 20210795N1. Cependant, cette rubrique apparaît grisée  dans le formulaire. Je vous joindrai la capture d''écran via Github.

Cordialement, 
Vincent Lipa

**Comment by Paris75012** [comment]
![image](https://user-images.githubusercontent.com/74540524/230304648-53f5db28-7e7d-40ac-be77-e9ce9bf2b5a9.png)

**Comment by SiropOps** [comment]
C'est normal; le spécimen est mise en collection; on ne peut renseigné les phases de propagations

**Comment by Paris75012** [comment]
Cyril,

C’est bien dommage que nous puissions pas indiquer ces informations après la mise en collection. Serait-il possible de formuler une demande d’évolution à ce sujet ?

Je te remercie pour la célérité de ta réponse,

Bien à toi,

Vincent Lipa
Responsable de la graineterie du JBP

***@***.***

Jardin botanique de Paris – Paris.fr<https://www.google.fr/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwiYgMnC0qXaAhXIkywKHcbyAL4QFgguMAA&url=https%3A%2F%2Fwww.paris.fr%2Fjardinbotaniquedeparis&usg=AOvVaw108ymee_CTrfV65b7K05nw>

Bureau :
Ville de Paris
Direction des Espaces Verts et de l'Environnement
Services des Techniques du Végétal et de l’Agriculture
Jardin Botanique
103 avenue de  France
75639 PARIS Cedex 13
Fax : + 33 (0)1 71 28 57 41

Graineterie du Jardin botanique de Paris :
Pavillon 5 - Parc Floral  75012 Paris
Tél. : + 33 (0)1 49 57 94 47

De : Cyril Boillat ***@***.***
Envoyé : lundi 24 avril 2023 15:00
À : Computaceae-IT/cit-ticketing ***@***.***>
Cc : Lipa, Vincent ***@***.***>; Comment ***@***.***>
Objet : Re: [Computaceae-IT/cit-ticketing] [CULT] BOTA-SC : rubrique propagation grisée (Issue #1232)

* Ce message a ete emis par un expediteur externe *
Prenez garde aux liens et aux pieces jointes. Ne fournissez jamais votre mot de passe de connexion Ville de Paris.

________________________________

C'est normal; le spécimen est mise en collection; on ne peut renseigné les phases de propagations

**Comment by SiropOps** [comment]
Oui, je te propose de faire un ticket de demande d'évolution et je le soumettrai à prochain vote de la CT.Bien à toi, CyrilEnvoyé depuis mon téléphone ☎️ 
-------- Message d'origine --------De : Paris75012 ***@***.***> Date : 24.04.23  15:24  (GMT+01:00) À : Computaceae-IT/cit-ticketing ***@***.***> Cc : Cyril Boillat ***@***.***>, State change ***@***.***> Objet : Re: [Computaceae-IT/cit-ticketing] [CULT] BOTA-SC : rubrique propagation grisée (Issue #1232) 
Cyril,



C’est bien dommage que nous puissions pas indiquer ces informations après la mise en collection. Serait-il possible de formuler une demande d’évolution à ce sujet ?



Je te remercie pour la célérité de ta réponse,



Bien à toi,



Vincent Lipa

Responsable de la graineterie du JBP



***@***.***



Jardin botanique de Paris – Paris.fr<https://www.google.fr/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwiYgMnC0qXaAhXIkywKHcbyAL4QFgguMAA&url=https%3A%2F%2Fwww.paris.fr%2Fjardinbotaniquedeparis&usg=AOvVaw108ymee_CTrfV65b7K05nw>



Bureau :

Ville de Paris

Direction des Espaces Verts et de l'Environnement

Services des Techniques du Végétal et de l’Agriculture

Jardin Botanique

103 avenue de  France

75639 PARIS Cedex 13

Fax : + 33 (0)1 71 28 57 41



Graineterie du Jardin botanique de Paris :

Pavillon 5 - Parc Floral  75012 Paris

Tél. : + 33 (0)1 49 57 94 47



De : Cyril Boillat ***@***.***

Envoyé : lundi 24 avril 2023 15:00

À : Computaceae-IT/cit-ticketing ***@***.***>

Cc : Lipa, Vincent ***@***.***>; Comment ***@***.***>

Objet : Re: [Computaceae-IT/cit-ticketing] [CULT] BOTA-SC : rubrique propagation grisée (Issue #1232)



* Ce message a ete emis par un expediteur externe *

Prenez garde aux liens et aux pieces jointes. Ne fournissez jamais votre mot de passe de connexion Ville de Paris.



________________________________



C'est normal; le spécimen est mise en collection; on ne peut renseigné les phases de propagations

**Comment by Paris75012** [comment]
C’est noté.

Bien Cordialement

Vincent Lipa
Responsable de la graineterie du JBP

***@***.***

Jardin botanique de Paris – Paris.fr<https://www.google.fr/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwiYgMnC0qXaAhXIkywKHcbyAL4QFgguMAA&url=https%3A%2F%2Fwww.paris.fr%2Fjardinbotaniquedeparis&usg=AOvVaw108ymee_CTrfV65b7K05nw>

Bureau :
Ville de Paris
Direction des Espaces Verts et de l'Environnement
Services des Techniques du Végétal et de l’Agriculture
Jardin Botanique
103 avenue de  France
75639 PARIS Cedex 13
Fax : + 33 (0)1 71 28 57 41

Graineterie du Jardin botanique de Paris :
Pavillon 5 - Parc Floral  75012 Paris
Tél. : + 33 (0)1 49 57 94 47

De : Cyril Boillat ***@***.***
Envoyé : lundi 24 avril 2023 15:37
À : Computaceae-IT/cit-ticketing ***@***.***>
Cc : Lipa, Vincent ***@***.***>; Comment ***@***.***>
Objet : Re: [Computaceae-IT/cit-ticketing] [CULT] BOTA-SC : rubrique propagation grisée (Issue #1232)

* Ce message a ete emis par un expediteur externe *
Prenez garde aux liens et aux pieces jointes. Ne fournissez jamais votre mot de passe de connexion Ville de Paris.

________________________________
Oui, je te propose de faire un ticket de demande d'évolution et je le soumettrai à prochain vote de la CT.Bien à toi, CyrilEnvoyé depuis mon téléphone ☎️
-------- Message d'origine --------De : Paris75012 ***@***.***<mailto:***@***.***>> Date : 24.04.23 15:24 (GMT+01:00) À : Computaceae-IT/cit-ticketing ***@***.***<mailto:***@***.***>> Cc : Cyril Boillat ***@***.***<mailto:***@***.***>>, State change ***@***.***<mailto:***@***.***>> Objet : Re: [Computaceae-IT/cit-ticketing] [CULT] BOTA-SC : rubrique propagation grisée (Issue #1232)
Cyril,



C’est bien dommage que nous puissions pas indiquer ces informations après la mise en collection. Serait-il possible de formuler une demande d’évolution à ce sujet ?



Je te remercie pour la célérité de ta réponse,



Bien à toi,



Vincent Lipa

Responsable de la graineterie du JBP



***@***.***<mailto:***@***.***>



Jardin botanique de Paris – Paris.fr<https://www.google.fr/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwiYgMnC0qXaAhXIkywKHcbyAL4QFgguMAA&url=https%3A%2F%2Fwww.paris.fr%2Fjardinbotaniquedeparis&usg=AOvVaw108ymee_CTrfV65b7K05nw>



Bureau :

Ville de Paris

Direction des Espaces Verts et de l'Environnement

Services des Techniques du Végétal et de l’Agriculture

Jardin Botanique

103 avenue de France

75639 PARIS Cedex 13

Fax : + 33 (0)1 71 28 57 41



Graineterie du Jardin botanique de Paris :

Pavillon 5 - Parc Floral 75012 Paris

Tél. : + 33 (0)1 49 57 94 47



De : Cyril Boillat ***@***.***<mailto:***@***.***>

Envoyé : lundi 24 avril 2023 15:00

À : Computaceae-IT/cit-ticketing ***@***.***<mailto:***@***.***>>

Cc : Lipa, Vincent ***@***.***<mailto:***@***.***>>; Comment ***@***.***<mailto:***@***.***>>

Objet : Re: [Computaceae-IT/cit-ticketing] [CULT] BOTA-SC : rubrique propagation grisée (Issue #1232)



* Ce message a ete emis par un expediteur externe *

Prenez garde aux liens et aux pieces jointes. Ne fournissez jamais votre mot de passe de connexion Ville de Paris.



________________________________



C'est normal; le spécimen est mise en collection; on ne peut renseigné les phases de propagations
